src/components/DatePicker/_DatePicker.scss

```diff
@@ -5,14 +5,22 @@ fieldset {
 
   &.datePicker {
     @include fieldset-reset;
+    margin-bottom: var(--space-m);
   }
 }
 
 .datePicker {
+  margin-bottom: var(--space-m);
+
   .sr-only {
     @include screenreader-only();
   }
 
+  .form-row,
+  .form-field {
+    margin-bottom: var(--space-xxs);
```

Comment:
You can set this to `margin-bottom: 0`.

Reply:
Shouldn't we use css variables?

Reply:
In this case, we want to kill the margin that is being added by `.form-row`/`.form-field`.  If we had `--space-none`, we could use that, but I don't think that is necessary.  

However, as I think about his more, the `.form-row`/`.form-field` classes are supposed to remove the bottom margin with they are `last-child`.  I will look at this again and see if the CSS in `Form` needs to be tweaked instead.

Reply:
Yes, the styles in `Form` need to be updated.  For now, set this to 0.